pylint_django/transforms/foreignkey.py

```diff
@@ -87,10 +100,24 @@ def infer_key_classes(node, context=None):
                 module_name = current_module.name
             elif not module_name.endswith('models'):
                 # otherwise Django allows specifying an app name first, e.g.
-                # ForeignKey('auth.User') so we try to convert that to
-                # 'auth.models', 'User' which works nicely with the `endswith()`
-                # comparison below
-                module_name += '.models'
+                # ForeignKey('auth.User')
+
+                django_spec = find_spec('django')
+                use_django_model_resolution = bool(django_spec)
```

Comment:
is this making sure that Django is installed ? If yes, then you don't need it b/c there's checkers/django_installed.py which will produce a fatal failure if you try executing pylint-django in environment which doesn't have Django.

Reply:
Yeah this is just a check to see if Django is installed. I didn't know about that check. I'll remove this.